sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/Attachment.kt

```diff
@@ -1,5 +1,7 @@
 package io.sentry.kotlin.multiplatform
 
+@Suppress("UnusedPrivateMember")
```

Comment:
Usually those annotations just pollute the API, would be possible to suppress via a generated file? for example, https://github.com/getsentry/sentry-dart/blob/main/flutter/config/detekt-bl.xml